ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -295,31 +295,31 @@ Apply DataScienceCluster CustomResource
 Create DataScienceCluster CustomResource Using Test Variables
     [Documentation]
     [Arguments]    ${dsc_name}=${DSC_NAME}
-    ${file_path} =    Set Variable    tasks/Resources/Files/
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
```

Comment:
an improvement could be to make this a Suite variable rather than repeating it

Reply:
Ok I will refactor it